frontend/src/components/navigation/AppNavBar.test.tsx

```diff
@@ -0,0 +1,325 @@
+import React from 'react';
```

Comment:
Is there a better way to organize tests?  This is getting to be a long file.

Reply:
Not that I know of, but you can group tests by functionality if you want. 
Ex) 
```
describe('nav bar link functionality', () => {
  it('related test 1', () => {   
  }
  it('related test 2', () => {   
  }
})
```